vue/test/components/ripe-pickers/ripe-pickers.test.js

```diff
@@ -1,19 +1,158 @@
+                            black: { available: true },
+                            blue: { available: true },
+                            brown: { available: true },
+                            red: { available: true }
+                        }
+                    },
+                    suede_cbe: {
+                        available: true,
+                        colors: {
+                            blue: { available: true },
+                            fuchsia: { available: true },
+                            red: { available: true }
+                        }
+                    }
+                }
+            },
+            top0_bottom: {
+                available: true,
+                materials: {
+                    crocodile_cbe: {
+                        available: true,
+                        colors: { black: { available: true }, chestnut: { available: true } }
+                    },
+                    leather_cbe: {
+                        available: true,
+                        colors: {
+                            black: { available: true },
+                            blue: { available: true },
+                            brown: { available: true },
+                            red: { available: true }
+                        }
+                    },
+                    suede_cbe: {
+                        available: true,
+                        colors: {
+                            blue: { available: true },
+                            fuchsia: { available: true },
+                            red: { available: true }
+                        }
+                    }
+                }
+            },
+            patch: {
+                available: true,
+                materials: {
+                    patch_cbe: {
+                        available: true,
+                        colors: { smiley: { available: true }, egg: { available: true } }
+                    }
+                }
+            }
+        };
+        this.options = {
+            parts: {
+                side: {
+                    color: "black",
+                    material: "crocodile_cbe",
+                    face: "side"
+                },
+                shadow: {
+                    color: "default",
+                    hidden: true,
+                    material: "default"
+                },
+                top0_bottom: {
+                    color: "fuchsia",
+                    face: "side",
+                    material: "suede_cbe"
+                }
             }
-        });
+        };
+    }
+
+    isReady() {}
+    bind() {}
+}
+
+describe("RipePickers", () => {
+    it("should instantiate the component", async () => {
+        const ripeInstance = new MockRipeSdk();
+        const component = await base.getComponent("RipePickers", { props: { ripe: ripeInstance } });
 
         assert.strictEqual(component.emitted("loading").length, 1);
+
+        await component.vm.$forceUpdate();

```

Comment:
Why is this needed?

Reply:
Because the `pre_config` and `post_config` events aren't binded in the test, so no component update. We need to manually do it ourselves.

Reply:
`$nextTick` would also do the trick

Reply:
What do you mean with "aren't binded"? The solution may end up being nextTick, but I really need more detail before settling with that.

Reply:
The parts, materials and colors values for the selects are computed, so we need to trigger something that can lead to a component rerender.

Reply:
Isn't there a more high-level way of doing this? E.g.: triggering an event, mutating the store...

Reply:
I don't think so because it's a vue-test issue